modules/odc-creating-apiserversource.adoc

```diff
@@ -7,9 +7,16 @@
 
 .Procedure
 
-. Navigate to the **Add** page and select **Event Source**.
-. In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-. Configure the **ApiServerSource** settings:
+. In the **Developer** perspective, navigate to **+Add** → **Event Source**. The  **Event Sources** page in the **Developer Catalog** is displayed.
```

Comment:
The  **Event Sources** page is displayed.

Reply:
Done.